graph/dijkstra.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * @function dijkstra
+ * @description Compute the shortest path from a source node to all other nodes. The input graph is an adjacency matrix where `0` represents no edge, and a positive value represents the weight of the edge. All weights are positive.
+ * @Complexity_Analysis
+ * Time complexity: O(V^2)
+ * Space Complexity: O(V)
+ * @param {number[][]} graph - The adjacency matrix graph
+ * @param {number} start - The source node
+ * @return {number[]} - The shortest path to each node
+ * @see https://en.wikipedia.org/wiki/Dijkstra%27s_algorithm
+ */
+export const dijkstra = (graph: number[][], start: number): number[] => {
+  let visited = Array(graph.length).fill(false);
+  let paths = Array(graph.length).fill(Number.MAX_SAFE_INTEGER);
```

Comment:
Why not use float infinity here? These seem to be distances rather than paths, so why not call them that?